mentat/commands.py

```diff
@@ -387,3 +388,31 @@ def argument_names(cls) -> list[str]:
     @classmethod
     def help_message(cls) -> str:
         return "Set a configuration option or omit value to see current value."
+
+
+class ScreenshotCommand(Command, command_name="screenshot"):
+    async def apply(self, *args: str) -> None:
+        if not args:
+            raise UserError("Screenshot command requires a URL or string to capture.")
```

Comment:
Let's not raise a UserError; these should be used to exit mentat with an error message; in this case, we just want to fail the command